testpilot/frontend/static-src/app/main.js

```diff
@@ -79,6 +79,15 @@ app.extend({
     } else {
       hitCallback();
     }
+  },
+
+  // Subscribe to basket
+  subscribeToBasket(email) {
+    const url = 'https://basket.mozilla.org/news/subscribe/';
+    const params = 'newsletters=test-pilot&email=' + email;
+    const request = new XMLHttpRequest();
+    request.open('POST', url);
+    request.send(params);
```

Comment:
Yeah, this should probably use the `fetch()` API. It wraps XMLHttpRequest in a nicer API. There's native support in a bunch of browsers, and we've imported a polyfill for the ones that don't have support.